modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/view_calendar.jsp

```diff
@@ -81,7 +81,9 @@ boolean columnOptionsVisible = GetterUtil.getBoolean(SessionClicks.get(request,
 	});
 </aui:script>
 
-<aui:container cssClass="calendar-portlet-column-parent">
+<clay:container
```

Comment:
Have we checked this change is correct?

```html
<div class="container-fluid calendar-portlet-column-parent">
```

```html
<div class="container-fluid container-fluid-max-xl calendar-portlet-column-parent />
```

Based on the rest, I assume so, just double checking :)

Reply:
I think that `.container-fluid-max-xl` is never a problem in our cases because it simply adds a `max-width` value of `1248px` to the container. So unless we absolutely need a width above the defined value we can always keep this class

https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/atlas/variables/_globals.scss#L25